code/_globalvars/lists/names.dm

```diff
@@ -14,7 +14,7 @@ var/list/mime_names = file2list("config/names/mime.txt")
 var/list/carp_names = file2list("config/names/carp.txt")
 var/list/golem_names = file2list("config/names/golem.txt")
 var/list/plasmaman_names = file2list("config/names/plasmaman.txt")
-var/list/posibrain_names = list("PBU","HIU","SINA","ARMA","OSI","HBL","MSO","RR","CHRI","CDB","HG","XSI","ORNG","GUN","KOR","MET","FRE","XIS","SLI","PKP","HOG","RZH","GOOF","MRPR","JJR","FIRC","INC","PHL","BGB","ANTR","MIW","WJ","JRD","CHOC","ANCL","JLLO","JNLG","KOS","TKRG","XAL","STLP","CBOS","DUNC","FXMC","DRSD")
+var/list/posibrain_names = list("PBU","HIU","SINA","ARMA","OSI","HBL","MSO","RR","CHRI","CDB","HG","XSI","ORNG","GUN","KOR","MET","FRE","XIS","SLI","PKP","HOG","RZH","GOOF","MRPR","JJR","FIRC","INC","PHL","BGB","ANTR","MIW","WJ","JRD","CHOC","ANCL","JLLO","JNLG","KOS","TKRG","XAL","STLP","CBOS","DUNC","FXMC","DRSD","COI", "AX")

```

Comment:
given they're so far apart, I'm inclined to believe it's:
* A not a reference to him (but what other hog do we have? hmm)
* Someone adding one for him as thanks without realising he had one
* Adding one for himself without realising he had one